src/generic/eigen_solver.cc

```diff
@@ -623,7 +641,7 @@ namespace oomph
     int info = 0;
 
     // Call FORTRAN LAPACK to get the required workspace
-    // Note the use of the padding flag
+    // Note the use of the padding leading dimension for the matrices
```

Comment:
Not clear where the padding leading dimension is used in the call. n is the original dimension isn't it?

Reply:
n is the size of the system which is indeed unchanged, the leading dimension follows the pointers to the matrices and is the padded one.